components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -70,18 +87,61 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
 
     // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattcharacteristic-getdescriptor
     fn GetDescriptor(&self) -> Option<Root<BluetoothRemoteGATTDescriptor>> {
-        //UNIMPLEMENTED
-        None
+        let (sender, receiver) = ipc::channel().unwrap();
+        self.global().r().as_window().bluetooth_thread().send(
```

Comment:
If we want change this code in the future, maybe write FIXME or TODO above this line ?